lerobot/configs/default.yaml

```diff
@@ -120,7 +120,9 @@ eval:
   # `batch_size` specifies the number of environments to use in a gym.vector.VectorEnv.
   batch_size: 1
   # `use_async_envs` specifies whether to use asynchronous environments (multiprocessing).
-  use_async_envs: false
+  use_async_envs: true
+  episode_progbar: true
```

Comment:
Mind just putting these two in the train script directly hardcoded as function call params? I don't know that we want to bother with adding more config params for something so inconsequential.

Reply:
Out of curiosity, why do we want to make the progress bar an option?